src/mecca/reactor/platform/linux/poller.d

```diff
@@ -1,7 +1,10 @@
-module mecca.reactor.platform.linux.epoll;
+module mecca.reactor.platform.linux.poller;
```

Comment:
This rename is weird -- since it is Linux-specific, so is indeed `epoll` based?

Reply:
To reduce the number of version blocks it's preferred to have as many types as possible with the same name on all platforms. Therefore a generic name like `Poller` is better than `Epoll`. I think it would look weird to have a struct named `Poller` in a module named `epoll`.

The reason for the `Kqueue` name is because it's out side the platform specific packages, since it's available on *BSD platforms as well. Then it's publicly imported into `mecca.reactor.platform.darwin.poller`. If Mecca gets support for a *BSD platform as well, it can reuse the `Kqueue` module.